### ISSUE TITLE
Add a Stata Quarto tutorial

### ISSUE THREAD

**Issue by hugetim** [hugetim/nbstata#37]
Quarto is blowing my mind. It seemed impenetrable as I browsed randomly through the reference docs to find answers to particular questions. But then I finally broke down and...worked through the tutorials. 🌩💡 

Working through the tutorials will be a bigger challenge for Stata users without experience in any of the tutorial reference languages, so I'd like to add a Stata Quarto tutorial to the docs here soon.

_Originally posted by @hugetim in https://github.com/hugetim/nbstata/discussions/11#discussioncomment-5273306_

**Comment by hugetim** [comment]
Can just link to this from the docs: https://friosavila.github.io/playingwithstata/quarto/dynstat

**Comment by friosavila** [comment]
Actually not only that one.
my whole side uses nbstata combined with Quarto
https://friosavila.github.io/stataviz/stataviz5.html

**Comment by friosavila** [comment]
Although a proper example would be useful

**Comment by gaksaray** [comment]
Any ideas on how to run Stata Quarto, but in Visual Editor of RStudio? The goal is to be able to work in Visual Editor as usual:

![rstudio-hello](https://github.com/hugetim/nbstata/assets/72267919/023984bf-e9bf-429f-ab74-c19effffec46)

but write in `{stata}` code chunks instead of `{r}` code chunks. Currently when I do that, I can render the document in Quarto successfully with Stata code and output. But, I cannot run individual code chunks written in Stata to show within the editor without rendering the whole document.

Is this possible?

**Comment by friosavila** [comment]
I do t think that is possible
Your only alternative option I would say is to use jupyter notebooks that
can later be transferred to quarto

**Comment by hugetim** [comment]
Adding a note that working with Stata in VSCode is now supported: https://github.com/hugetim/nbstata/discussions/44#discussion-6499362

**Comment by hugetim** [comment]
Right, I think that is true as of Jan 1, 2024. https://github.com/quarto-dev/quarto/issues/336#issuecomment-1873437072

Here's how it works for me, just now:
![image](https://github.com/user-attachments/assets/3a4cd4e7-2b4f-4eff-96f0-d41073f67634)

That is, I have to manually change the kernel to nbstata (by clicking the upper left text, under "Interactive-1").

**Comment by friosavila** [comment]
We are doing the same then
Now if we could get the option of inline Stata…
